src/constants/wallets.ts

```diff
@@ -352,15 +352,5 @@ export const DYDX_CHAIN_INFO: Parameters<typeof suggestChain>[0] = {
 // TODO: export this type from abacus instead
 export enum DydxChainAsset {
   USDC = 'usdc',
-  DYDX = 'dydx',
+  CHAIN = 'chain',
```

Comment:
mega-nit: Instead of `DydxChainAsset.CHAIN`, do you think that `DydxChainAsset.NATIVE` sounds better?